content/kubermatic/master/advanced/presets/_index.en.md

```diff
@@ -11,87 +11,108 @@ With Presets you can specify default settings for new Cluster. Use Presets to re
 
 ### Core concept
 
-As a Kubermatic administrator with superuser access, you can define Presets type in a standard format using a YAML
-definition, allowing the assignment of new credential types to supported providers. This allows you to define a custom
-credential type that works in ways similar to existing credential types. For example, you could create a custom credential
-type that injects access keys, passwords or network settings into Cloud object.
+As a Kubermatic administrator with superuser access, you can define Presets type in a standard format using static YAML
+file or Kubernetes Custom Resource Definition (CRD) struct that represents the Preset, allowing the assignment of new
+credential types to supported providers. This allows you to define a custom credential type that works in ways similar
+to existing credential types. For example, you could create a custom credential type that injects access keys, passwords
+or network settings into Cloud object.
 
-User can specify credential list with unique names for each provider. The API allows using only credential name and never exposes the credential values.
+Users can specify a credential list with unique names for the group of providers. This credential set can be used for every
+logged in user or can be filtered out by email domain.
+ 
+The API allows using only credential names and never exposes the credential values.
 The proper credential name is used for credential injection.
 
 If the Preset name is used together with standard credentials the preset is taken as a first.
 
 ### Prerequisites
 
-A presets are optional for Kubermatic API. The Kubermatic API takes a flag:
+A presets are optional for Kubermatic API. The Kubermatic API takes a flags:
```

Comment:
Presets are optional for the Kubermatic API. The Kubermatic API takes the flags:

In best case, a customer might never see the flags of the Kubermatic API service. Maybe better reference to the Helm values.

Reply:
good point, I've added this.